db/schema.rb

```diff
@@ -12,42 +10,43 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20_201_029_155_643) do
+ActiveRecord::Schema.define(version: 2020_10_29_155643) do
+
   # These are extensions that must be enabled in order to support this database
-  enable_extension 'plpgsql'
-
-  create_table 'course_students', force: :cascade do |t|
-    t.bigint 'course_id'
-    t.bigint 'student_id'
-    t.integer 'points'
-    t.index ['course_id'], name: 'index_course_students_on_course_id'
-    t.index ['student_id'], name: 'index_course_students_on_student_id'
+  enable_extension "plpgsql"
+
+  create_table "course_students", force: :cascade do |t|
+    t.bigint "course_id"
+    t.bigint "student_id"
+    t.integer "points"
+    t.index ["course_id"], name: "index_course_students_on_course_id"
+    t.index ["student_id"], name: "index_course_students_on_student_id"
   end
 
-  create_table 'courses', force: :cascade do |t|
-    t.string 'name'
-    t.integer 'course_code'
-    t.string 'school_name'
-    t.bigint 'teacher_id'
-    t.index ['teacher_id'], name: 'index_courses_on_teacher_id'
+  create_table "courses", force: :cascade do |t|
+    t.string "name"
+    t.integer "course_code"
+    t.string "school_name"
+    t.bigint "teacher_id"
+    t.index ["teacher_id"], name: "index_courses_on_teacher_id"
   end
 
-  create_table 'poms', force: :cascade do |t|
-    t.string 'name'
-    t.string 'description'
-    t.string 'duration'
-    t.string 'link'
+  create_table "poms", force: :cascade do |t|
+    t.string "name"
+    t.string "description"
+    t.string "duration"
+    t.string "link"
   end
 
-  create_table 'students', force: :cascade do |t|
-    t.string 'name'
+  create_table "students", force: :cascade do |t|
+    t.string "name"
```

Comment:
This will be a merge conflict moving forward, as part of our work today involved updating both students and teachers to have attributes. This was part of @Arique1104 and @ckccameron s branch around oauth_setup.

We can resolve later.

Reply:
do you mean because of the double quotes rather than single? That can be changed to whatever the other branches have

Reply:
No, he means that there will be a merge conflict because our student now requires more than just name.  I see what you're saying @dhalverson, once we make a pull request, @ckccameron and I will sift through the merge conflicts.  Thanks for the heads up!